src/components/LoginForm/LoginForm.jsx

```diff
@@ -1,25 +1,32 @@
+/* eslint-disable no-nested-ternary */
```

Comment:
What is the main idea of that PR? Add Eslint settings or customisation of login page?

Reply:
removed